src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1584,6 +1590,15 @@ bool esbmc_parseoptionst::process_goto_program(
       goto_k_induction(goto_functions, msg);
     }
 
+    if(cmdline.isset("goto-contractor"))
+    {
+      #ifdef ENABLE_GOTO_CONTRACTOR
+      goto_contractort(goto_functions, msg);
+      #else
+      msg.error("Current build does not support contractors");
+      #endif
+    }
```

Comment:
define indentation needs to be fixed here

Reply:
fixed.